src/backend/columnar/columnar_customscan.c

```diff
@@ -655,18 +1033,77 @@ ColumnarScan_CreateCustomScanState(CustomScan *cscan)
 	CustomScanState *cscanstate = &columnarScanState->custom_scanstate;
 	cscanstate->methods = &ColumnarScanExecuteMethods;
 
-	if (EnableColumnarQualPushdown)
+	return (Node *) cscanstate;
+}
+
+
```

Comment:
minor: I'm not a fan of this convention but I think we need a simple comment here, like:
```c
/*
 * EvalParamsMutator foo bar something ....
 */
```

Reply:
Done.